lib/spack/spack/package.py

```diff
@@ -67,7 +67,6 @@
 from spack.build_systems.waf import WafPackage
 from spack.build_systems.xorg import XorgPackage
 from spack.builder import run_after, run_before
-from spack.dependency import all_deptypes
```

Comment:
Maybe this shouldn't be done? :thinking: probably breaking. but confusing that it was exported to be honest.